packages/commons-server/src/libs/server/server.ts

```diff
@@ -630,6 +745,150 @@ export class MockoonServer extends (EventEmitter as new () => TypedEmitter<Serve
     }
   }
 
+  private sendFileWithCallback(
+    route: Route,
+    routeResponse: RouteResponse,
+    callback: Callback,
+    invocation: CallbackInvocation,
+    request: Request,
+    response: Response
+  ) {
+    if (!callback.filePath) {
+      return;
+    }
+
+    const fileServingError = (error) => {
+      this.emit('error', ServerErrorCodes.ROUTE_FILE_SERVING_ERROR, error);
```

Comment:
Please create a new error code. Something like CALLBACK_FILE_ERROR, otherwise it will be logged and display a toast in the app like a regular response related file error. 
Like the CALLBACK_ERROR, please include a payload containing the `callbackName` when emitting. 

(same remark for the same error emitted in this function)